designs/PackageReference-Extern-Alias.md

```diff
@@ -0,0 +1,164 @@
+# Extern alias support with PackageReference
+
+* Status: In Review
+* Author(s): [Nikolche Kolev](https://github.com/nkolev92)
+* Issue: [4989](https://github.com/NuGet/Home/issues/4989) Extern alias support for NuGet package references
+* Type: Feature
+
+## Problem Background
+
+In the PackageReference world there is no way to add assembly specific metadata.
+Sometimes multiple packages might carry the same type, and in those cases the solution is to use the alias feature. However, there's no native way to do that in PackageReference.
+In comparison, in packages.config, one can add the extra metadata to the assembly included because all the assembly references are written in the project file. 
+
+## Who are the customers
+
+All PackageReference customers
+
+## Requirements
+
+* A feature or guidance to allow the customers to specify an alias for an assembly.

```

Comment:
Was there a embed interop type version of this? I can't remember? If not, can we just bite these both off at the same time.

Reply:
There is!
That one is an author driven feature.
But hunting through the issues, I don't think it was fully implemented :(
As in NuGet did the work, but the build tasks are not consuming it so not very useful :(

Original issue: 
https://github.com/NuGet/Home/issues/2365

2 tasks that need completed:
https://github.com/dotnet/NuGet.BuildTasks/issues/69
https://github.com/dotnet/sdk/issues/2732

//cc @aortiz-msft We should have someone from NuGet ensure the partner changes are handled.